packages/fury-adapter-swagger/package.json

```diff
@@ -1,11 +1,13 @@
 {
   "name": "fury-adapter-swagger",
-  "version": "0.27.1",
+  "version": "0.27.2",
   "description": "Swagger 2.0 parser for Fury.js",
   "author": "Apiary.io <support@apiary.io>",
   "license": "MIT",
   "main": "./lib/adapter.js",
-  "files": ["lib/*.js"],
+  "files": [
```

Comment:
NPM/YARN reformatted file, will stick with the format the tools use so we don't have to fight it every time we do a release.